webpack.config.js

```diff
@@ -45,6 +45,9 @@ var config = {
   resolve: {
     root: path.resolve('./src'),
     extensions: ['', '.js']
+  },
+  externals: {
+    pouchdb: 'PouchDB'
```

Comment:
@kosssi it may be a typo here, cf. https://github.com/cozy/cozy-client-js/pull/43